sphinx/builders/latex/__init__.py

```diff
@@ -228,6 +230,7 @@ def write(self, *ignored) -> None:
 
         for entry in self.document_data:
             docname, targetname, title, author, docclass = entry[:5]
+            theme = self.themes.get(docclass)
```

Comment:
Now we're using themes only for generating .tex files. Addition to this, it should contain additional files (ex. .sty file). So I must write a handler for coping theme-related files.